go.mod

```diff
@@ -1,19 +1,21 @@
 module github.com/open-cluster-management/insights-client
 
-go 1.15
+go 1.16
 
 require (
-	cloud.google.com/go v0.54.0 // indirect
-	github.com/Azure/go-autorest/autorest v0.11.1 // indirect
-	github.com/Azure/go-autorest/autorest/adal v0.9.5 // indirect
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/golangci/golangci-lint v1.24.0 // indirect
```

Comment:
removed lint from require? Was that intentional, or the result of a `go mod build` operation

Reply:
intentional because it's only used by the build lint step and not the product.